### DISCUSSION TITLE
App design

### DISCUSSION THREAD

**Discussion by YuriDevAT** [YurisCodingClub/pro-check#8] [Ideas]
How should the design of the app look like?
Let's collect some ideas (maybe from [Behance](https://www.behance.net/) or [dribbble](https://dribbble.com/)) and choose with which kind of design we want to go further.
 
- [ ] Design for the form components
- [ ] Design for page layout (components)

**Comment by eduarf** [comment]


**Reply by YuriDevAT** [reply]
Hi @eduarf. Thank you for your comment. I am sorry that it was not clear from my instructions on which kind of app ProCheck will be in the end. The developer should be able to easily go through a checklist to make dure they don't miss any steps when showcasing their projects. An example can be found in the project's wiki.

**Reply by YuriDevAT** [reply]
It's all good 👍

**Comment by YuriDevAT** [comment]
I found some design examples I really like 😄 

1. Suggestions no1. [Minimalist dark mode](https://dribbble.com/shots/14630755-Cards-Dark-UI)
<img width="235" alt="Suggestion No1 dark mode" src="https://user-images.githubusercontent.com/54622834/195993251-4de64c22-6223-4424-907b-a6eab9a8ca9d.png">

2. Adding[ a progress bar like in this example](https://dribbble.com/shots/9841628-Get-started-Cheklist) could be a nice feature
<img width="313" alt="" src="https://user-images.githubusercontent.com/54622834/195993382-3fb03f45-57e7-4fb7-bfce-00a12d7e6f43.png">

3. This could also be nice, [having both lists next to each other](https://dribbble.com/shots/15461831-call-center-feedback-kanban).
<img width="493" alt="" src="https://user-images.githubusercontent.com/54622834/195993427-7a74216c-6742-4235-860f-2016b3b1a399.png">
